src/gatsby/models/blog/createPages.js

```diff
@@ -47,41 +47,25 @@ function* pagesGenerator({ itemCount, hasHeroItem = false, basePath }) {
 }
 
 const createPages = async ({ graphql, actions }) => {
+  const isDevMode = process.env.NODE_ENV === 'development'
```

Comment:
there should be already a check like this around the code, should be a helper function